src/routes/PetPage/PetPage.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Section } from '../../components/Utils/Utils';
+import PetsService from '../../services/petsService';
+import { Link } from 'react-router-dom';
+
+export default function PetPage () {
+  const params = useParams();
+  const [pet, setPet] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (!pet) {
+      PetsService.getPet(params.petID)
+        .then(pet => setPet(pet))
+        .catch(e => {
+          setError(e.error);
+        });
+    }
+  });
+
+  const renderFavoriteIcon = () => {
+    // TODO
+    if (true) {
+      return <span className='text-danger'><FontAwesomeIcon icon='heart' /></span>;
+    }
+    return <span className='text-danger'><FontAwesomeIcon icon={['far', 'heart']} /></span>;
+  }
+
+  const renderPetBio = () => {
+    return (
+      <div className='col-md m-1'>
+        <h2>
+          Hi, I'm {pet.Name}!
+          <span> </span>
+          {renderFavoriteIcon()}
+          </h2>
+        <table className='table table-sm table-borderless'>
+          <tbody>
+            <tr>
+              <td scope='row'>Breed:</td>
```

Comment:
In the case where the animal type is "Other", do you think we should change this from "Breed" to "Species" or something?

A way that might address this comment and the one above is to ditch the label and just have a line under the name that says [Type], [Breed]. So a beagle might say "Dog, Beagle", and a gerbil might say "Other, Gerbil". Thoughts?

Reply:
I think that's a great idea! I can make it like a subtitle